README.md

```diff
@@ -99,7 +99,7 @@ compileOnlyDependencies.extendsFrom(configurations.getByName("compileOnly"))
 
 dependencies {
   add(shadowedDependencies.name, "com.squareup.okhttp3:okhttp:4.9.0")
-  add(compileOnlyDependencies.name, gradleApi())
+  add("compileOnly", gradleApi())
```

Comment:
could this be simplified?
```suggestion
  compileOnly(gradleApi())
```

Reply:
I kind of like it as it is because it works without generated accessors (and also [aligns nicely](https://xkcd.com/276/) with the line above)